plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -133,6 +134,24 @@
     group: super_group/mon_group
     access_level: owner
   delegate_to: localhost
+
+- name: "Block GitLab User"
+  gitlab_user:
+    api_url: https://gitlab.example.com/
+    api_token: "{{ access_token }}"
+    validate_certs: False
+    username: myusername
+    state: blocked
+  delegate_to: localhost
```

Comment:
```suggestion
```
Will it work without ```delegate_to``` ? If yes, IMO this is an extra piece of info and we can remove them from all examples

Reply:
Yes, it works. But as the module has a dependency on the python-gitlab library, it may be better to keep the delegation local. It is off course up to the user of the module to decide what they want.
So yes, `delegate_to` can be removed from the examples